### PR TITLE
Introduction + Homepage Branch

### PR DESCRIPTION
**Pull request by tejas-muthya37**
This branch includes screens of the Welcome page (index.html and styles.css) and the Homepage.

### CODE REVIEW

**Comment by romabulani** [comment]
The code looks good. here also commi this file /path/to/highlight.min.js, as I was not able to view the page

**Comment by tejas-muthya37** [comment]
@romabulani , this is just a CDN for highlight.js. Cannot commit that file as it is not physically present on my machine, I've just added a reference.

PS - highlight.js is a script used to show pieces of code to our webpage.

**Review by romabulani — APPROVED** [review]
Ok Understood

File: Homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    align-content: center;
+    margin-bottom: 3rem;
+    height: max-content;
+}
+
+.header-logo {
+    font-family: 'Gochi Hand', cursive;
```
Comment (Parul1999) [inline Homepage/homepage.css]: Font family , define in main class and use it as css variables

File: Homepage/homepage.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Magic - CSS Component Library</title>
+    <link href="homepage.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```
Comment (Parul1999) [inline Homepage/homepage.html]: Suggestion - You can import font/icon links in the css file itself which u are linking here.

File: Homepage/homepage.css (hunk 1 above)
Comment (tejas-muthya37) [reply Homepage/homepage.css]: Changes done and pushed. Thanks.

**Review by Parul1999 — APPROVED** [review]

**Pull request merged**